README.md

```diff
@@ -67,13 +92,13 @@ If you are on certain distribution of Linux, you will need to alias `fdfind` to
 
 ### Using custom key bindings
 
-Each function is assigned mnemonic key bindings by default (see screenshots above) in [conf.d/fzf.fish][]. However, if you would like to customize them, you can prevent the default key bindings from executing by setting `fzf_fish_custom_keybindings` as a [universal variable][]. You can do this by running
+Each function is assigned mnemonic key bindings by default (see [Features](#Features)). However, if you would like to customize them, you can prevent the default key bindings from executing by setting `fzf_fish_custom_keybindings` as a [universal variable][]. You can do this by running
```

Comment:
Now that there is a section for features, link to that instead of referring to screenshots above